### PR TITLE
Hpos instrumentation

### PR DESCRIPTION
**Pull request by yashgit891**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: includes/plugin-instrumentation.php [hunk 1]
```diff
@@ -30,11 +30,43 @@ function razorpayPluginActivated()
...
+            $properties = [
+                'isHposEnabled' => true
+            ];
+
+            $response = $this->rzpTrackSegment('hpos.interacted', $properties);
+            $this->rzpTrackDataLake('hpos.interacted', $properties);
+
+            update_option('rzp_hpos', 'yes');
+        }
+        else if(OrderUtil::custom_orders_table_usage_is_enabled() === false and
```
Comment (abdulwahidsharief) [inline includes/plugin-instrumentation.php:55]: @yashgit891 can we use else here

File: includes/plugin-instrumentation.php [hunk 2]
```diff
@@ -30,11 +30,43 @@ function razorpayPluginActivated()
...
+                empty(get_option('rzp_hpos')) === false and
```
Comment (abdulwahidsharief) [inline includes/plugin-instrumentation.php:56]: get_option('rzp_hpos') === 'yes' condition will do job of empty(get_option('rzp_hpos')) === false. a condition can be removed

File: includes/plugin-instrumentation.php (hunk 1 above)
Comment (yashgit891) [reply includes/plugin-instrumentation.php:55]: @abdulwahidsharief no as it would lead to false positives.

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**